liquibase-couchbase/src/main/java/liquibase/ext/couchbase/validator/MutateInLongValueValidator.java

```diff
@@ -17,13 +15,9 @@ public MutateInLongValueValidator(MutateInType mutateInType) {
     }
 
     @Override
-    public void validate(String path, Value value, List<Value> values) {
+    public void validate(String path, List<Value> values) {
         validatePathPresence(path);
-        validateValuePresence(value);
-        validateNoMultipleValues(values);
-        if (value.getType() != DataType.LONG) {
-            throw new MutateInDataTypeNotAllowedException(value.getType(), mutateInType);
-        }
+        validateMultipleValuesPresence(values);
```

Comment:
Again like in the comment above I think we break logic here. We need to ensure that only 1 value is passed and that it's LONG type. Here you just remove validation on LONG, we need it


You need to create method like validateOneValue where you will check that list contains only 1 value

And check that this value is LONG type (not in the new method above)